### PR TITLE
Updated the installation to use npx

### PR DESCRIPTION
**Pull request by woodyrew**
npx ships with npm as of v5.2.0

Before submitting a pull request, make sure it's targeting the right branch:

- For fixes to Ink 1.0, use `master`.
- For fixes to Foundation for Emails 2, use `v2.0`.

Happy coding! :)

### CODE REVIEW

File: docs/pages/sass-guide.md [hunk 1]
```diff
@@ -28,22 +28,22 @@ To use the Sass version of Foundation for Emails, you need Node.js installed on
 
 ## Installing
 
-We'll use the Foundation CLI to set up a new project. If you already have the Foundation CLI installed from Foundation for Sites, you can skip this first command.
+We'll use [npx](https://www.npmjs.com/package/npx) to run the Foundation CLI to set up a new project. If you have npm version [https://github.com/npm/npm/releases/tag/v5.2.0](https://github.com/npm/npm/releases/tag/v5.2.0) or later, you already have npx, you can skip this first command.
 
 ```bash
-npm install --global foundation-cli
+npm install --global npx
```
Comment (DanielRuf) [inline docs/pages/sass-guide.md]: This is not really needed anymore.

File: docs/pages/sass-guide.md [hunk 2]
```diff
@@ -28,22 +28,22 @@ To use the Sass version of Foundation for Emails, you need Node.js installed on
...
 ```
 
 If you run into any permission errors (EACCESS) on OS X or Linux, you can try prefixing the command with `sudo`.
 
 ```bash
-sudo npm install --global foundation-cli
+sudo npm install --global npx
```
Comment (DanielRuf) [inline docs/pages/sass-guide.md]: Not needed.

**Review by DanielRuf — CHANGES_REQUESTED** [review]

**Comment by woodyrew** [comment]
Thanks for the review @DanielRuf. I've made changes as requested.

File: docs/pages/sass-guide.md [hunk 3]
```diff
@@ -52,7 +40,13 @@ The CLI will ask you for a project name, which is used as the name of the folder
 
 ## Running the Server
 
-After your project has been installed, run `cd project`, where `project` is the name of the project just created. Then run:
+After your project has been installed, run `cd project`, where `project` is the name of the project just created. Install the dependancies:
```
Comment (DanielRuf) [inline docs/pages/sass-guide.md]: dependancies => dependencies

**Review by DanielRuf — CHANGES_REQUESTED** [review]

File: docs/pages/sass-guide.md (hunk 3 above)
Comment (woodyrew) [reply docs/pages/sass-guide.md]: Fixed

File: docs/pages/sass-guide.md [hunk 4]
```diff
@@ -28,22 +28,10 @@ To use the Sass version of Foundation for Emails, you need Node.js installed on
 
 ## Installing
 
-We'll use the Foundation CLI to set up a new project. If you already have the Foundation CLI installed from Foundation for Sites, you can skip this first command.
+Run the Foundation CLI as a one-off command using [npx](https://www.npmjs.com/package/npx), it’s super easy to fire up a blank Foundation for Emails project. Move into the folder you store your projects in, and then run this command:
```
Comment (DanielRuf) [inline docs/pages/sass-guide.md]:
```suggestion
Run the Foundation CLI as a one-off command using [npx](https://www.npmjs.com/package/npx), it's super easy to fire up a blank Foundation for Emails project. Move into the folder you store your projects in, and then run this command:
```

**Review by DanielRuf — CHANGES_REQUESTED** [review]

**Comment by DanielRuf** [comment]
In general I would also keep the old part as this is also an option (for those wo can not or do not want to use npx and want to install it globally.

So please a the old part too. The recommended way is npx and yarn.

**Comment by DanielRuf** [comment]
> So please add the old part too. The recommended way is npx and yarn.

**Comment by woodyrew** [comment]
Sounds like you have a good idea how you'd like it to be conveyed. Feel free to use this PR as a starting point. :smile:

**Pull request closed without merging**